src/components/WatchLater.jsx

```diff
@@ -0,0 +1,42 @@
+import { useSelector, useDispatch } from 'react-redux'
+import { Link } from 'react-router-dom'
+import watchLaterSlice from '../data/watchLaterSlice'
+import Movie from './Movie'
+import '../styles/starred.scss'
+
+const WatchLater = ({viewTrailer}) => {
+
+    const state = useSelector((state) => state)
+    const { watchLater } = state
+    const { remveAllWatchLater } = watchLaterSlice.actions
```

Comment:
It looks like there's a typo in the remveAllWatchLater variable declaration in the watchLaterSlice.actions. Perhaps it was meant to be removeAllWatchLater instead?

By the way, using an auto-typo-checking tool within our coding environment could help catch these kinds of issues early on. It might be a good idea to explore integrating such a tool to streamline our development process and catch these small errors before they cause any trouble.